src/app/screens/nftCollection/index.tsx

```diff
@@ -183,7 +188,14 @@ function NftCollection() {
                   key={nft.asset_identifier}
                   item={nft}
                   itemId={getNftCollectionsGridItemId(nft, collectionData)}
-                  onClick={handleOnClick}
+                  onClick={() => {
+                    if (isBnsCollection(nft.asset_identifier)) return;
+                    if (nft.data?.token_metadata) {
+                      navigate(`/nft-dashboard/nft-detail/${nft.data?.fully_qualified_token_id}`);
+                    } else {
+                      toast.custom(ToastContent);
+                    }
+                  }}
```

Comment:
if it's bns collection, we want the onClick handler to be undefined, so that the cursor CSS does not show a pointer

Reply:
oh makes sense, added the fix